features/268-next-cf/api.md

```diff
@@ -614,3 +614,53 @@ Currently, micro-service allow developers to do routing in the app, while others
 - Custom Domains Support #246 (https://github.com/SkygearIO/features/issues/246)
 
 Handled by the k8s ingress controller. Skycli may provide command for skygear user to add these features.
+
+## Secret management
+
+With immutable deployment, secrets are also immutable. For managing secrets,
+following operations are allowed.
+
+- skycli secret create
+- skycli secret remove
+- skycli secret rename
+
+To avoid human error, deployment will be failed if config file is using secrets
+that are not exist. Removing or renaming a secret will not affect existing deployment.
```

Comment:
> Removing or renaming a secret will not affect existing deployment.

Want to confirming about rename secret.

1. After rename secret, all existing deployment keeps no change, a developer may need to update each `secrets` section of a `cf` in `skygear.yaml` and run deploy again to use the secret by the updated name.
2. When a developer run `skycli secret list`, he cannot see secrets with the previous name.